### PR TITLE
Fixed the issue of the data/bss section cannot be read from ARM FVP d…

### PR DESCRIPTION
**Pull request by yajunxiaMS**
…ebug tool in cortex-A7 GNU port.

https://msazure.visualstudio.com/One/_workitems/edit/24597276/

## PR checklist

- [ x ] Updated function header with a short description and version number
- [ x ] Added test case for bug fix or new feature
- [ x ] Validated on real hardware

### CODE REVIEW

File: ports/cortex_a7/gnu/example_build/sample_threadx.out
Comment (TiejunMS) [inline ports/cortex_a7/gnu/example_build/sample_threadx.out:1]: This file should not be added.

File: ports/cortex_a7/gnu/example_build/test.log
Comment (TiejunMS) [inline ports/cortex_a7/gnu/example_build/test.log:1]: Is this file for test purpose?

**Review by TiejunMS — APPROVED** [review]

**Review by xiuwencai — APPROVED** [review]

File: ports/cortex_a7/gnu/example_build/sample_threadx.out
Comment (yajunxiaMS) [reply ports/cortex_a7/gnu/example_build/sample_threadx.out:1]: Removed, :)

File: ports/cortex_a12/gnu/example_build/sample_threadx.ld [hunk 1]
```diff
@@ -109,7 +109,7 @@ SECTIONS
   .eh_frame_hdr : { *(.eh_frame_hdr) }
   /* Adjust the address for the data segment.  We want to adjust up to
      the same address within the page on the next page up.  */
-  . = ALIGN(256) + (. & (256 - 1));
+  . = 0x2E000000;
```
Comment (xiaocq2001) [inline ports/cortex_a12/gnu/example_build/sample_threadx.ld:112]: Seems it's a fixed address, is that fit for all OEM chips or just for specific one?

Comment (yajunxiaMS) [reply ports/cortex_a12/gnu/example_build/sample_threadx.ld:112]: yes, its fixed address:
Refer to:
https://developer.arm.com/documentation/100964/1123/Versatile-Express-Model/VE-memory-map-for-Cortex-A-series?lang=en

**Pull request merged**